Resources/Prototypes/Species/reptilian.yml

```
@@ -56,13 +56,13 @@
       points: 1
       required: false
     Chest:
-      points: 3
+      points: 4 # imp
       required: false
     Legs:
-      points: 2
+      points: 8 # imp
```

```suggestion
      points: 8 # imp 2<8
```
Note original value for future maintenance work.